tests/specs/graph/jsr/FastCheck_Vars.txt

```diff
@@ -148,16 +148,14 @@ Fast check https://jsr.io/@scope/a/1.0.0/mod.ts:
       ]
     ]
   };
-  export const func1 = function(): string {
-    return {} as never;
-  };
-  export const func2 = (): string =>({} as never);
-  export const func3 = (): string =>({} as never);
-  export const func4 = ()=>"test" as const;
-  export const func5 = (): Promise<string> =>({} as never);
-  export const func6 = (): Promise<1> =>({} as never);
-  export const func7 = (): Promise<1> =>({} as never);
-  export const func8 = async ()=>"test" as const;
+  export const func1: () => string = {} as never;
+  export const func2: () => string = {} as never;
+  export const func3: () => string = {} as never;
+  export const func4: () => "test" = {} as never;
+  export const func5: () => Promise<string> = {} as never;
+  export const func6: () => Promise<1> = {} as never;
+  export const func7: () => Promise<1> = {} as never;
+  export const func8: () => Promise<"test"> = {} as never;
```

Comment:
As part of this change, let's revert this and get the output more like before in order to focus the change a bit more. I need to spend some time investigating the reason why we don't do this OOTB and to document it. I remember there was a good reason (maybe not for this specific case), but I'm pre-occupied at the moment to investigate (will get to it soon). Then we can follow up later with this change in a separate PR.